packages/relay/src/lib/eth.ts

```diff
@@ -1016,6 +1063,41 @@ export class EthImpl implements Eth {
     });
   }
 
+  /**
+   * returns the block response  
+   * otherwise return undefined.
+   *
+   * @param blockNumberOrTag
+   * @param returnLatest
+   */
+  private async getHistoricalBlockResponse(blockNumberOrTag?: string | null, returnLatest?: boolean): Promise<any | null> {
+    let blockResponse: any;
+    // Determine if the latest block should be returned and if not then just return null
+    if (!returnLatest && 
+      (blockNumberOrTag == null || blockNumberOrTag == 'latest' || blockNumberOrTag == 'pending')) {
```

Comment:
```suggestion
      (blockNumberOrTag == null || blockNumberOrTag === 'latest' || blockNumberOrTag === 'pending')) {
```

Reply:
nit: Also pull 'latest' and `pending` and `'earliest` out into class static constants.